packages/mantine/src/components/button/ButtonWithDisabledTooltip.tsx

```diff
@@ -1,58 +1,37 @@
-import {Box, Tooltip} from '@mantine/core';
-import {Property} from 'csstype';
-import {forwardRef, MouseEventHandler} from 'react';
+import {Box, Tooltip, TooltipProps} from '@mantine/core';
+import {forwardRef, ReactNode} from 'react';
 
 import {createPolymorphicComponent} from '../../utils';
 
 export interface ButtonWithDisabledTooltipProps {
-    disabled?: boolean;
-    onClick?: MouseEventHandler<HTMLButtonElement>;
     /**
      * The tooltip message to display when disabled
      */
     disabledTooltip?: string;
     /**
-     * The background color when disabled
-     *
-     * @default 'unset'
+     * Whether the button underneath the tooltip is disabled
+     */
+    disabled?: boolean;
+    children?: ReactNode;
+    /**
+     * Additional tooltip props to set on the disabled button tooltip
      */
-    disabledHoverColor?: Property.BackgroundColor;
+    disabledTooltipProps?: Omit<TooltipProps, 'disabled' | 'label' | 'children'>;
```

Comment:
This is useful to set a specific width on long tooltips so that they wrap on multiple lines. I had a few cases where I had to put this in the admin:

`disabledTooltipProps={{width: 300}}`